supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemJobExecutionActor.scala

```diff
@@ -57,6 +69,16 @@ class SharedFileSystemJobExecutionActor(override val jobDescriptor: BackendJobDe
     completionPromise.future
   }
 
+  override def recover: Future[BackendJobExecutionResponse] = {
+    val executorRef = context.actorOf(asyncPropsCreator(completionPromise), "SharedFileSystemAsyncJobExecutionActor")
```

Comment:
Not that I'm aware of. This is only ever called once by each sync actor I believe, either by executer _or_ recover. If the sync actor _did_ try to create more than one async actors, I would be interested to find out why.